src/main/java/githubstudy/Main.kt

```diff
@@ -3,6 +3,8 @@ package githubstudy
 object Main {
     @JvmStatic
     fun main(args: Array<String>) {
-        println("Hello world!!")
+        for (i in 1..10) {
+            println("Hello world!!")
```

Comment:
I like one bang instead of two bangs!

```suggestion
            println("Hello world!")
```

Reply:
Is that so?